xbmc/network/WebServer.cpp

```diff
@@ -102,9 +102,23 @@ CWebServer::CWebServer()
 #endif
 }
 
+static MHD_Response* create_response(size_t size, void* data, int free, int copy)
+{
+#if (MHD_VERSION >= 0x00094001)
+  MHD_ResponseMemoryMode mode = MHD_RESPMEM_PERSISTENT;
+  if (copy)
+    mode = MHD_RESPMEM_MUST_COPY;
+  else if (free)
+    mode = MHD_RESPMEM_MUST_FREE;
+  return MHD_create_response_from_buffer(0, nullptr, mode);
+#else
+  return MHD_create_response_from_data(0, nullptr, free, copy);
```

Comment:
Sure. I'll PR when @MilhouseVH confirms it is good (I suggested both calls are fixed).